lightning/src/ln/functional_tests.rs

```diff
@@ -7743,9 +7744,13 @@ fn test_bump_penalty_txn_on_revoked_htlcs() {
 	let nodes = create_network(2, &node_cfgs, &node_chanmgrs);
 
 	let chan = create_announced_chan_between_nodes_with_value(&nodes, 0, 1, 1000000, 59000000, InitFeatures::known(), InitFeatures::known());
-	// Lock HTLC in both directions
-	let payment_preimage = route_payment(&nodes[0], &vec!(&nodes[1])[..], 3_000_000).0;
-	route_payment(&nodes[1], &vec!(&nodes[0])[..], 3_000_000).0;
+	// Lock HTLC in both directions (using a slightly lower CLTV delay to provide timely RBF bumps)
+	let route = get_route(&nodes[0].node.get_our_node_id(), &nodes[0].net_graph_msg_handler.network_graph.read().unwrap(),
+		&nodes[1].node.get_our_node_id(), Some(InvoiceFeatures::known()), None, &Vec::new(), 3_000_000, 50, nodes[0].logger).unwrap();
+	let payment_preimage = send_along_route(&nodes[0], route, &[&nodes[1]], 3_000_000).0;
+	let route = get_route(&nodes[1].node.get_our_node_id(), &nodes[1].net_graph_msg_handler.network_graph.read().unwrap(),
+		&nodes[0].node.get_our_node_id(), Some(InvoiceFeatures::known()), None, &Vec::new(), 3_000_000, 50, nodes[0].logger).unwrap();
```

Comment:
Could you use an expression using `TEST_FINAL_CLTV` in replace of the `50` magic number so that the calculation is clear?

Reply:
No - this test relies on the CLTV diff being 50 as it assumes the RBF bump will happen at certain block heights. When `TEST_FINAL_CLTV` was increased, the test suddenly fails because RBF bumps don't happen when they're "supposed" to.

Reply:
Not sure I follow. I'm just suggesting replacing `50` with `TEST_FINAL_CLTV - 20` or `TEST_FINAL_CLTV - SOME_CONST_DESCRIBING_THE_DIFF`. Otherwise, it isn't clear where `50` is coming from.

Reply:
Ah, could define a new constant for it, but using `TEST_FINAL_CLTV` would be incorrect here as, were `TEST_FINAL_CLTV` to change again, this test would then be buggy/wrong. The test relies on exactly 50, irrespective of what other CLTV settings/constants are set.

Reply:
Gotcha. So how was 50 calculated?

Reply:
I think its more the other way around - the CLTV delay *was* 50, and the test was written assuming the CLTV diff is 50 and expecting RBF bumps on the timeline of an HTLC that times out in 50 blocks. It could be anything, but it requires rewriting the test.

Reply:
Ah, I guess my confusion came from the fact that `route_payment` uses `TEST_FINAL_CLTV` and the updated comment in this test says, "(using a slightly lower CLTV delay to provide timely RBF bumps)". So I figured you just wanted `route_payment` with a different CLTV.